joomla/joomla-cms
# [4.4] Set permission for add-module-to-dashboard button

Pull Request for Issue #41624 .

PR #37268 set wrong permission for the "Add module to dahboard" Button on cpanel. 
This was caused because a user had a special use case - descibed in https://github.com/joomla/joomla-cms/issues/37267

### Summary of Changes
This PR sets the permission back to core.manage on com_modules. 

Therefore the issue https://github.com/joomla/joomla-cms/issues/37267 is unresolved again. 
This special use case needs an other solution. Maybe an override of cpanel. 

### Testing Instructions
You need a user in usergroup manager. Login as manager - you see the button "Add Module to the dashboard". 
Activate. 
An Error message is displayed.

Apply the patch - no button for usergroup "manager". 



### Link to documentations
Please select:
